docs/userguide/modules/getting-started/partials/connect-petclinic-pgcluster.adoc

```diff
@@ -0,0 +1,114 @@
+== Connect the application to the database
+
+Suppose the {servicebinding-title} is not present. In that case, the
+application\'s admin needs to extract all the configuration details and
+create a Secret resource and expose it to the application through volume
+mount in Kubernetes. The steps would be something like this:
+
+. Identify the required values for connecting the application to the
+database
```

Comment:
```suggestion
. Identify the required values for connecting the application to the
database.
```
Period at the end of a complete sentence.

Reply:
Thx